### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by krevetkos**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,159 @@
+* {
+  box-sizing: border-box; }
+
+body {
+  margin: 0;
+  margin-left: auto;
+  margin-right: auto;
```
Comment (karmeljuk) [inline css/style.css]: margin shorthand

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  display: flex;
+  flex-direction: column;
+  margin: 5px 0; }
+  .info-wraper .info-container {
+    display: flex;
+    flex-direction: column; }
+    .info-wraper .info-container .info-item {
+      border: 1px solid gray;
+      margin-left: auto;
+      margin-right: auto;
```
Comment (karmeljuk) [inline css/style.css]: margin shorthand

File: css/style.css (hunk 1 above)
Comment (krevetkos) [reply css/style.css]: This is what the scss mixin did, i'll fix it.

Comment (karmeljuk) [reply css/style.css]: ok

**Pull request merged**